src/.editorconfig

```diff
@@ -0,0 +1,4 @@
+﻿[*.cs]
```

Comment:
I don't think there is much value in adding an `.editorconfig` file here because we have a separate task which covers setting all this up (#12 ). If you are going to add one, please copy it from https://github.com/buerokratt/DMR/blob/main/src/Dmr.Api/.editorconfig and also take note of the docs in this PR: https://github.com/buerokratt/Cross-functional-requirements/pull/40

Reply:
Honestly not sure how I managed to generate this, will replace with the proper .editorconfig